documentation/cookbook/working-with-test-suite.markdown

```diff
@@ -346,14 +199,34 @@ public function testWholeWorld(){}
 This is very handy if you're trying to fix or write one particular set of test methods and need to execute it
 again and again.
 
+## Automatic code checks ##
+
+If you plan to create a pull request, your code must pass automatic code checks. 
 
-## Fix Coding Style ##
+### Stan and Psalm ###
 
-You can fix your coding style __before creating your commit__ using [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer)
-and the provided project configuration.
+Run static analyzers [stan](https://phpstan.org/) and [psalm](https://psalm.dev/) through composer scripts:
+```bash
+composer run-script stan
+composer run-script psalm
+```
+
+### Code style ###
+
+You can fix code style before creating a PR using [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) and the provided project configuration.
+
+```bash
+composer run-script cs-fix # automatically adjusts formatting
```

Comment:
composer cs-fix 
etc suffices

Reply:
oh man, and I have been typing it out for years like an idiot lol